src/app/components/queue-v2/queues-v2.component.ts

```diff
@@ -159,6 +159,13 @@ function queueVisualization(rawData : QueueInfo){
           .attr("height", 30)
           .attr("fill", "#e6f4ea")
           .attr("class", "cardBottom");
+        
+        group.append("image")
+        .attr("href", "./assets/images/hierarchy.svg") 
+        .attr("x", 5) 
+        .attr("y", 5) 
+        .attr("width", 20)
+        .attr("height", 20);
```

Comment:
```suggestion
        group.append("image")
          .attr("href", "./assets/images/hierarchy.svg") 
          .attr("x", 5) 
          .attr("y", 5) 
          .attr("width", 20)
          .attr("height", 20);
```

Reply:
Should we care about the indent like other group here?

Reply:
Thanks @baconYao.